controllers/token.go

```diff
@@ -275,10 +275,14 @@ func (c *ApiController) IntrospectToken() {
 	tokenValue := c.Input().Get("token")
 	clientId, clientSecret, ok := c.Ctx.Request.BasicAuth()
 	if !ok {
-		util.LogWarning(c.Ctx, "Basic Authorization parses failed")
-		c.Data["json"] = Response{Status: "error", Msg: "Unauthorized operation"}
-		c.ServeJSON()
-		return
+		clientId = c.Input().Get("client_id")
+		clientSecret = c.Input().Get("client_secret")
+		if clientId == "" || clientSecret == "" {
+			util.LogWarning(c.Ctx, "Basic Authorization parses failed")
+			c.Data["json"] = Response{Status: "error", Msg: "Unauthorized operation"}
+			c.ServeJSON()
```

Comment:
Use `c.ResponseError()`

Don't use logWarning, just return the error in ResponseErrror

![image](https://user-images.githubusercontent.com/3787410/164352519-41015791-1baa-4971-85c0-5090ed3b9bc8.png)